kit/postbuild.sh

```diff
@@ -3,3 +3,7 @@ set -euo pipefail
 IFS=$'\n\t'
 
 cp src/routes/_toctree.yml build/_toctree.yml
+
+# To void conflict with Hub Tailwind CSS build, 
+# making doc-builder's PostCSS geenrated file an empty one
+> $(find . -regex '.*build.*.css')
```

Comment:
technically, making the CSS file an empty file rather than deleting it to avoid `resouce not found errors`.